src/pages/MySlayer.tsx

```diff
@@ -77,17 +122,43 @@ const MySlayerPage = () => {
                       />
                     );
                   })}
+                  {/* STAKED SLAYERS */}
+                  {stakedTokens.map(
```

Comment:
If the variable would be called `stakedSlayers`, you can remove the comment and let the code speak for itself. I think `stakedTokens` is good too, just trying to point out that naming can replace comments :)